packages/base/source/server/events.js

```diff
@@ -5,7 +5,7 @@ Space.messaging.define(Space.messaging.Event, 'Todos', {
   },
 
   TodoCreated: {
-    id: Guid,
+    todoId: Guid,
```

Comment:
My point it that it is *never bad to be explicit* and then we have a consistent pattern for referencing ids of other entities as the `sourceId`. 
So i would favor this style:
https://github.com/meteor-space/donations/blob/develop/packages/base/source/server/appeals/events.js#L7 over having to figure out the context of `id` by parsing and understanding the name of the event :wink: